src/a2a/server/request_handlers/rest_handler.py

```diff
@@ -264,12 +265,12 @@ async def on_get_task(
             return MessageToDict(proto_utils.ToProto.task(task))
         raise ServerError(error=TaskNotFoundError())
 
-    async def list_push_notifications(
+    async def list_tasks(
```

Comment:
nit: Can you re-order these methods to follow the pre-existing order? First `list_push_notifications` then `list_tasks`